kubejobs/__init__.py

```diff
@@ -79,7 +77,6 @@ def __repr__(self):
             "starting_time": str(self.get_application_start_time()),
             "status": self.status,
             "visualizer_url": self.visualizer_url,
-            "execution_time": self.execution_time,
```

Comment:
As you're removing this attribute from the Job entity, shouldn't you update the docs for this? https://github.com/ufcg-lsd/asperathos-manager/blob/master/docs/restapi-endpoints.md

Reply:
Agreed